_contrib/travis-check-png-optimized.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/sh
+
+## Finds newly added or modified PNG files in the repository and checks if each file is optimized.
+## If not optimized, it fails with an error code of 1
+
+CHANGEDFILES=$(git diff --name-only --diff-filter=AM "$TRAVIS_COMMIT_RANGE")
```

Comment:
Maybe instead of `$TRAVIS_COMMIT_RANGE` use `$(git merge-base master HEAD)` so this is easier to run on our own systems.

Reply:
Good point. I'm addressing this in an upcoming push.